src/index.html

```diff
@@ -4,10 +4,33 @@
     <meta charset="UTF-8">
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
     <title>Product cards</title>
     <link rel="stylesheet" href="./styles/index.scss">
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div class="card" data-qa="card">
+      <img src="images/imac.jpeg"
+      alt="product" class="card__photo">
```

Comment:
This one still not fixed, if you have 3 or more attributes then each of them should be in new line with two more spaces from left in relation to opening tag and closing tag or bracket should be with the same amount of spaces as opening tag

```suggestion
      <img
        src="images/imac.jpeg"
        alt="product"
        class="card__photo"
      >
```